### PR TITLE
Restore displaying description for antd

### LINKED ISSUE #2273: "description" of a property won't show when using theme antd
### Description

[Example](https://rjsf-team.github.io/react-jsonschema-form/#eyJmb3JtRGF0YSI6eyJmaXJzdE5hbWUiOiJDaHVjayIsImxhc3ROYW1lIjoiTm9ycmlzIiwiYWdlIjo3NSwiYmlvIjoiUm91bmRob3VzZSBraWNraW5nIGFzc2VzIHNpbmNlIDE5NDAiLCJwYXNzd29yZCI6Im5vbmVlZCJ9LCJzY2hlbWEiOnsidGl0bGUiOiJBIHJlZ2lzdHJhdGlvbiBmb3JtIiwiZGVzY3JpcHRpb24iOiJUaGlzIGlzIHRoZSBzYW1lIGFzIHRoZSBzaW1wbGUgZm9ybSwgYnV0IGl0IGlzIHJlbmRlcmVkIGFzIGEgYm9vdHN0cmFwIGdyaWQuIFRyeSBzaHJpbmtpbmcgdGhlIGJyb3dzZXIgd2luZG93IHRvIHNlZSBpdCBpbiBhY3Rpb24uIiwidHlwZSI6Im9iamVjdCIsInJlcXVpcmVkIjpbImZpcnN0TmFtZSIsImxhc3ROYW1lIl0sInByb3BlcnRpZXMiOnsiZmlyc3ROYW1lIjp7InR5cGUiOiJzdHJpbmciLCJ0aXRsZSI6IkZpcnN0IG5hbWUiLCJkZXNjcmlwdGlvbiI6IlRoaXMgaXMgdGhlIHNhbWUgYXMgdGhlIHNpbXBsZSBmb3JtLCBidXQgaXQgaXMgcmVuZGVyZWQgYXMgYSBib290c3RyYXAgZ3JpZC4gVHJ5IHNocmlua2luZyB0aGUgYnJvd3NlciB3aW5kb3cgdG8gc2VlIGl0IGluIGFjdGlvbi4ifSwibGFzdE5hbWUiOnsidHlwZSI6InN0cmluZyIsInRpdGxlIjoiTGFzdCBuYW1lIn0sImFnZSI6eyJ0eXBlIjoiaW50ZWdlciIsInRpdGxlIjoiQWdlIn0sImJpbyI6eyJ0eXBlIjoic3RyaW5nIiwidGl0bGUiOiJCaW8ifSwicGFzc3dvcmQiOnsidHlwZSI6InN0cmluZyIsInRpdGxlIjoiUGFzc3dvcmQiLCJtaW5MZW5ndGgiOjN9LCJ0ZWxlcGhvbmUiOnsidHlwZSI6InN0cmluZyIsInRpdGxlIjoiVGVsZXBob25lIiwibWluTGVuZ3RoIjoxMH19fSwidWlTY2hlbWEiOnt9LCJ0aGVtZSI6ImFudGQiLCJsaXZlU2V0dGluZ3MiOnsidmFsaWRhdGUiOmZhbHNlLCJkaXNhYmxlIjpmYWxzZSwib21pdEV4dHJhRGF0YSI6ZmFsc2UsImxpdmVPbWl0IjpmYWxzZX19)
When using theme `@rjsf/antd`, `description` of a property won't show.

The problem will be resolved if I uncomment [this line](https://github.com/rjsf-team/react-jsonschema-form/blob/dafd55a01891634a62cf3b2ab9f1cf6a9a1ff26d/packages/antd/src/templates/FieldTemplate/index.js#L67). Is that commented out for some other reason?

### Steps to Reproduce

1. Add `description` for any property at [rjsf-jsonschema-form playground](https://rjsf-team.github.io/react-jsonschema-form/), like:
```diff
 "properties": {
    "firstName": {
      "type": "string",
      "title": "First name",
 +++     "description": "This is the same as the simple form, but it is rendered as a bootstrap grid. Try shrinking the browser window to see it in action."
    },
```
2. Switch the theme, and see the result.

> default
image

> antd
image

#### Expected behavior

`description` should be shown as what default theme did.

### PR DESCRIPTION
**Pull request by Ezhvsalate**
### Reasons for making this change

fixes #2273 

### Checklist

* [ ] **I'm updating documentation**
  - [ ] I've [checked the rendering](https://react-jsonschema-form.readthedocs.io/en/latest/#contributing) of the Markdown text I've added
* [ ] **I'm adding or updating code**
  - [ ] I've added and/or updated tests
  - [ ] I've updated [docs](https://react-jsonschema-form.readthedocs.io/) if needed
  - [ ] I've updated the [changelog](https://github.com/rjsf-team/react-jsonschema-form/blob/master/CHANGELOG.md) with a description of the PR
* [ ] **I'm adding a new feature**
  - [ ] I've updated the playground with an example use of the feature

### CODE REVIEW

**Review by epicfaace — COMMENTED** [review]
Can we add a snapshot test that exercises this feature?

File: packages/antd/src/templates/FieldTemplate/index.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ const FieldTemplate = ({
       ) : (
         <Form.Item
           colon={colon}
-          // extra={!!rawDescription && description}
+          extra={!!rawDescription && description}
```
Comment (epicfaace) [inline packages/antd/src/templates/FieldTemplate/index.js:67]: Do you know what the `!!rawDescription` is doing here?

Comment (Ezhvsalate) [reply packages/antd/src/templates/FieldTemplate/index.js:67]: As I see  `description` is an instance of `DescriptionField`, and `rawDescription` is just a plain text. So here we check whether `rawDescription` is not falsy and pass a `DescriptionField` to extra prop of antd Form.Item.
BTW, `DescriptionField` returns just `<span id={id}>{description}</span>`  for antd so nothing changes visually also if we just pass a `rawDescription` to `extra`.

Comment (Ezhvsalate) [reply packages/antd/src/templates/FieldTemplate/index.js]: Checked that and seems like it works without `rawDescription` well. Antd `form.item` handles null/undefined description without additional checks on our side.

Comment (epicfaace) [reply packages/antd/src/templates/FieldTemplate/index.js]: Thanks. Do you know what `rawDescription` is even used for then?? Is it ever used in other widgets / themes / elsewhere in the codebase?

**Comment by yuhanmcgee** [comment]
will this fix be merged soon? thanks

File: packages/antd/src/templates/FieldTemplate/index.js (hunk 1 above)
Comment (Ezhvsalate) [reply packages/antd/src/templates/FieldTemplate/index.js]: It is not used anywhere else in antd theme now. Possibly in some other themes you should pass a raw string, not react object to make it work. Anyway it looks like to be out of scope of this PR related to antd theme only.

File: packages/antd/test/Form.test.js [hunk 2]
```diff
@@ -94,4 +94,19 @@ describe("single fields", () => {
...
+          title: "test",
+          description: "test description",
+        }
+      }
+    };
+    const tree = renderer
+      .create(<Form schema={schema} />)
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
```
Comment (epicfaace) [inline packages/antd/test/Form.test.js:111]: One more thing. Can you add another test that sets `ui:description` and ensures that the description is also rendered there?

**Review by epicfaace — CHANGES_REQUESTED** [review]

File: packages/antd/test/Form.test.js (hunk 2 above)
Comment (ianwith) [reply packages/antd/test/Form.test.js:111]: @epicfaace I added the test here, could you please check? thanks! https://github.com/rjsf-team/react-jsonschema-form/pull/2644

**Comment by epicfaace** [comment]
Thanks for your help @Ezhvsalate ! Going to supersede this PR with https://github.com/rjsf-team/react-jsonschema-form/pull/2644.

**Pull request closed without merging**